docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -594,23 +957,43 @@ Start the second and the third nodes on two separate VMs as follows:
 After starting the yugabyted processes on all the nodes, configure the data placement constraint of the YugabyteDB cluster as follows:
 
 ```sh
-./bin/yugabyted configure --fault_tolerance=region
+./bin/yugabyted configure data_placement --fault_tolerance=region
 ```
 
 This command determines the data placement constraint based on the `--cloud_location` of each node in the cluster. If there are 3 or more regions available in the cluster, the command configures the cluster to survive at least 1 region failure. Otherwise, it outputs a warning message.
 
 The replication factor of the cluster defaults to 3.
 
-You can set the data placement constraint manually using the `--data_placement_constraint` flag, which takes the comma-separated value of `cloud.region.zone` as follows:
+You can set the data placement constraint manually using the `--constraint_value` flag, which takes the comma-separated value of `cloud.region.zone` as follows:
 
 ```sh
-./bin/yugabyted configure --fault_tolerance=region --data_placement_constraint=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a
+./bin/yugabyted configure data_placement --fault_tolerance=region --constraint_value=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a
 ```
 
 You can set the replication factor of the cluster manually using the `--rf` flag as follows:
 
 ```sh
-./bin/yugabyted configure --fault_tolerance=region --data_placement_constraint=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a --rf=3
+./bin/yugabyted configure data_placement --fault_tolerance=region --constraint_value=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a --rf=3
+```
+
+### Toggling Encryption-at-rest in a multi-zone or multi-region cluster
+
+{{< note title="Note" >}}
```

Comment:
This note doesn't need to be repeated twenty times. It should be moved up the doc structure to apply to multiple subsections